__tests__/integration/utils/wallet-helper.js

```diff
@@ -168,4 +281,86 @@ export class WalletHelper {
     }
     return transaction;
   }
+
+  /**
+   * Sends a transaction.
+   *
+   * @example
+   * wallet.sendTx({
+   *   destination: 'abc123',
+   *   value: 100,
+   *   token: 'def456'
+   * })
+   * @see https://wallet-headless.docs.hathor.network/#/paths/~1wallet~1simple-send-tx/post
+   * @param options
+   * @param {unknown} [options.fullObject] Advanced usage: a full body to send to post on 'send-tx'
+   * @param {{hash:string,index:number,token?:string}[]} [options.inputs] Optional Inputs
```

Comment:
I expected a user trying to make a more specific use case to use the advanced `fullObject` parameter. But for documentation's sake to a developer learning the code, it's nice to have this information here too.

Two `@typedef`s were added describing inputs and outputs with examples.

If we ever change this method's contract, this will be one more place to update the documentation, though.